cluster/azure/README.md

```diff
@@ -25,11 +26,10 @@ Bedrock provides different templates to start from when building your deployment
 
 The following templates are currently available for deployment:
 
-- [azure-common-infra](../environments/azure-common-infra): Common infrastructure deployment template.
-
-- [azure-simple](../environments/azure-simple/): Single cluster deployment.
-- [azure-single-keyvault](../environments/azure-single-keyvault): Single cluster with Azure Keyvault integration through flex volumes template.
-- [azure-multiple-clusters](../environments/azure-multiple-clusters/): Multiple cluster deployment with Traffic Manager.
+- [azure-common-infra](../environments/azure-common-infra): Common infrastructure deployment template, required to deploy azure-single-keyvault and azure-multiple-clusters.
+- [azure-simple](../environments/azure-simple/): Simple single cluster deployment.
+- [azure-single-keyvault](../environments/azure-single-keyvault): Single cluster with Azure Keyvault integration through flex volumes template. Requires [azure-common-infra](../environments/azure-common-infra) deployed first.
```

Comment:
Single Cluster with Azure Keyvault Integration - provided by the Kubernetes Keyvault FlexVolume component